gfx/qcms/src/transform.rs

```diff
@@ -56,6 +56,8 @@ use crate::{
 use std::sync::atomic::Ordering;
 use std::sync::Arc;
 
+use std::arch::is_aarch64_feature_detected;
```

Comment:
I have never had to add this to get Gecko building. Also I checked the code upstream and there is nothing like that as well. Taking into account the amount of platforms supported by Firefox I suspect it might be some issue in your setup

Reply:
OK I've just hit this one. Could you please upload the patch upstream?